modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomValidation.testcase

```diff
@@ -149,13 +149,20 @@ definition {
 	}
 
 	@description = "LPS-144474 - Verify that the Validation tab is available"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanViewValidationTab {
 		property portal.acceptance = "true";
 
-		// TODO LPS-153546 CanViewValidationTab pending implementation
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object 153546",
+			objectName = "CustomObject153546",
+			pluralLabelName = "Custom Objects 153546");
+
+		ObjectAdmin.openObjectAdmin();
+
+		ObjectPortlet.selectCustomObject(label = "Custom Object 153546");
 
+		ObjectAdmin.goToValidationsTab();
```

Comment:
Can you add an assertion to check portal is on the Validations tab? Any element that's unique to the Validations tab is fine. The test is clicking the Validations tab but we don't know if we're actually there.